source/basics.rst

```diff
@@ -92,11 +92,16 @@ It's generally good practice to include it, though it is not required.
 
 .. schema_example::
 
-    { "$schema": "http://json-schema.org/schema#" }
+    { "$schema": "http://json-schema.org/draft/2019-09/schema#" }
```

Comment:
In draft 2019-09 and later, it is preferred to not include a trailing `#` in the URI.